voting/src/voting.rs

```diff
@@ -151,6 +151,8 @@ impl Context {
                 let mut vote_box = self.meeting_mut().get_vote_box(box_key.as_slice()).unwrap();
                 vote_box.drop_in_box(vote.vote_id.clone());
 
+                self.meeting_mut().update_vote_box(&meeting_id, &vote_box);
```

Comment:
Please do not ignore returned Result type.
```suggestion
                self.meeting_mut().update_vote_box(&meeting_id, &vote_box).unwrap("We get the vote box before");
```